ci-operator/step-registry/aws/provision/localzone/aws-provision-localzone-commands.sh

```diff
@@ -0,0 +1,184 @@
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+function aws_add_param_to_json() {
+    local k="$1"
+    local v="$2"
+    local param_json="$3"
+    if [ ! -e "$param_json" ]; then
+        echo -n '[]' > "$param_json"
+    fi
+    cat <<< "$(jq  --arg k "$k" --arg v "$v" '. += [{"ParameterKey":$k, "ParameterValue":$v}]' "$param_json")" > "$param_json"
+}
+
+function aws_describe_stack() {
+    local aws_region=$1
+    local stack_name=$2
+    local output_json="$3"
+    cmd="aws --region ${aws_region} cloudformation describe-stacks --stack-name ${stack_name} > '${output_json}'"
+    run_command "${cmd}" &
+    wait "$!" || return 1
+    return 0
+}
+
+function aws_create_stack() {
+    local aws_region=$1
+    local stack_name=$2
+    local template_body="$3"
+    local parameters="$4"
+    local options="$5"
+    local output_json="$6"
+
+    cmd="aws --region ${aws_region} cloudformation create-stack --stack-name ${stack_name} ${options} --template-body '${template_body}' --parameters '${parameters}'"
+    run_command "${cmd}" &
+    wait "$!" || return 1
+
+    cmd="aws --region ${aws_region} cloudformation wait stack-create-complete --stack-name ${stack_name}"
+    run_command "${cmd}" &
+    wait "$!" || return 1
+
+    aws_describe_stack ${aws_region} ${stack_name} "$output_json" &
+    wait "$!" || return 1
+
+    return 0
+}
+
+localzone_subnet_tpl="/tmp/localzone_subnet_tpl.yaml"
+
+cat > ${localzone_subnet_tpl} << EOF
+# CloudFormation template used to create Local Zone subnets and dependencies
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for Best Practice VPC with 1-3 AZs
+
+Parameters:
+  ClusterName:
+    Description: ClusterName used to prefix resource names
+    Type: String
+  VpcId:
+    Description: VPC Id
+    Type: String
+  LocalZoneName:
+    Description: Local Zone Name (Example us-east-1-bos-1)
+    Type: String
+  RouteTableId:
+    Description: Route Table ID to associate the Local Zone subnet
+    Type: String
+  SubnetCidr:
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-4]))$
+    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-24.
+    Default: 10.0.128.0/20
+    Description: CIDR block for Subnet
+    Type: String
+
+Resources:
+  LocalZoneSubnet:
+    Type: "AWS::EC2::Subnet"
+    Properties:
+      VpcId: !Ref VpcId
+      CidrBlock: !Ref SubnetCidr
+      AvailabilityZone: !Ref LocalZoneName
+      Tags:
+      - Key: Name
+        Value: !Join
+          - ""
+          - [ !Ref ClusterName, !Ref LocalZoneName, "-1" ]
+      - Key: kubernetes.io/cluster/unmanaged
+        Value: "true"
+
+  SubnetRouteTableAssociation:
+    Type: "AWS::EC2::SubnetRouteTableAssociation"
+    Properties:
+      SubnetId: !Ref LocalZoneSubnet
+      RouteTableId: !Ref RouteTableId
+
+Outputs:
+  SubnetId:
+    Description: Subnet ID.
+    Value: !Ref LocalZoneSubnet
+EOF
```

Comment:
Why not getting from the installer repo. Some examples:

https://github.com/openshift/release/blob/eee207c1fb11c489da92ac88ab6fa87a9b04ce38/ci-operator/step-registry/ipi/conf/aws/sharednetwork/ipi-conf-aws-sharednetwork-commands.sh

Reply:
https://issues.redhat.com/browse/SPLAT-1028